Lombiq.Tests.UI/SecurityScanning/SecurityScanConfiguration.cs

```diff
@@ -279,6 +331,31 @@ internal async Task ApplyToPlanAsync(YamlDocument yamlDocument, UITestContext co
 #pragma warning disable S3878 // Arrays should not be created for params parameters
         yamlDocument.AddExcludePathsRegex([.. _excludedUrlRegexPatterns]);
 #pragma warning restore S3878 // Arrays should not be created for params parameters
+
+        if (AdminIsExcluded) yamlDocument.AddExcludePathsRegex($".*{context.AdminUrlPrefix}.*");
+
+        if (UnusedDatabaseTechnologiesAreExcluded)
+        {
+            var excludes = yamlDocument
+                .GetCurrentContext()
+                .GetOrAddNode<YamlMappingNode>("technology")
+                .GetOrAddNode<YamlSequenceNode>("exclude");
+
+            if (context.Configuration.UseSqlServer)
+            {
+                excludes.Add(new YamlScalarNode("MySQL"));
+                excludes.Add(new YamlScalarNode("PostgreSQL"));
+                excludes.Add(new YamlScalarNode("SQLite"));
+            }
+            else
+            {
+                // Assuming SQLite.
+                excludes.Add(new YamlScalarNode("Microsoft SQL Server"));
+                excludes.Add(new YamlScalarNode("MySQL"));
+                excludes.Add(new YamlScalarNode("PostgreSQL"));
```

Comment:
Note that `excludes` here is not a `List<T>` or anything but a `YamlSequenceNode`. That doesn't have `AddRange()`, we can only add nodes one by one. Thus if we were to put these two triplets of `YamlScalarNode` into some static, then we'd need to iterate through them and them one by one.

This would result in more and more complex code, which would most possibly also be slower (so we're talking about nanoseconds here in any case), with perhaps the advantage of fewer allocations if more than one security scan is run. I don't think it's worth it.